package.json

```diff
@@ -35,7 +35,6 @@
     "watch:server": "nodemon -e js,jsx --watch src/server src/client --exec 'npm run build:server && node src/server'"
   },
   "dependencies": {
```

Comment:
At the moment it seems to be assuming eslint installed globally.
I think the recommendation is to make it a project dependency but if not, maybe some instructions on the readme would be good